src/host/ut_host/ScreenBufferTests.cpp

```diff
@@ -3051,6 +3055,499 @@ void ScreenBufferTests::DontResetColorsAboveVirtualBottom()
     }
 }
 
+template<class T>
+void _FillLine(COORD position, T fillContent, TextAttribute fillAttr)
+{
+    auto& gci = ServiceLocator::LocateGlobals().getConsoleInformation();
+    auto& si = gci.GetActiveOutputBuffer().GetActiveBuffer();
+    auto& row = si.GetTextBuffer().GetRowByOffset(position.Y);
+    row.WriteCells({ fillContent, fillAttr }, position.X, false);
+}
+
+template<class T>
+void _FillLine(int line, T fillContent, TextAttribute fillAttr)
+{
+    _FillLine({ 0, SHORT(line) }, fillContent, fillAttr);

```

Comment:
nit: gsl narrow_cast this `SHORT`

Reply:
I think I've done this for all my casts now. Although I've used `gsl::narrow` rather than `narrow_cast`, since that seems to be the preference in these tests, and I'm assuming it's safer.